### DISCUSSION TITLE
Bug Report: Files viewed in Pull Request displays a negative number

### DISCUSSION THREAD

**Discussion by Vol0kin** [community/community#7312] [Pull Requests]
Hi there! Today I was reviewing a PR and I stumbled accross a very strange bug. I had previously reviewed all files and checked all of the boxes in the `Files changed` section of a PR. I exited out of it and when I came back, every single box was checked, but it displayed 0/10 files viewed. When I started unchecking them, it started displaying a negative value of files viewed (see attached screenshot for more details).

I don't know how this happened or what are the reproduction steps, since I haven't been able to reproduce it again. Perhaps it was caused by previously marking every file as viewed, modifying the PR with a new commit (merging the target branch at which the PR points, for example) and then comming back. Maybe when the target branch was merged it rest all of the viewed files to 0 but didn't uncheck them.

![github-files-viewed-bug](https://user-images.githubusercontent.com/26794767/140410354-3167f569-deea-40db-b208-1999b2e4ce29.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
